docs/UserGuide.md

```diff
@@ -193,6 +193,22 @@ Examples:
 * `deleteseller p/98765432` deletes the seller with phone number `98765432` from the client book.
 
     ![result for 'deleteseller p/98765432'](images/deleteseller.png)
+
+### Adding a property : `addproperty`
+
+Add a specified property into the client book.
+
+Format: `addproperty c/POSTAL_CODE u/UNIT_NUMBER`
+
+* Adds a property with the specified `POSTAL_CODE` and `UNIT_NUMBER`.
+* The `POSTAL_CODE` must be exactly 6 digits with each digit in the range [0-9]. It does not accept any non-integer characters or spaces
+* The `UNIT_NUMBER` comprises of numbers that is delimited by exactly one dash(-). On either side of the dash are numbers comprising of two or more digit. The range of numbers of the left hand side of the dash is [00-148] and the right hand side is [00-111110]. Other than the dash bo other non-integer characters or spaces are accepted
```

Comment:
There's a spelling error "bo" but ig u can change it in ur next pull? or I can rmbr to change it for u in mine. And also small thing but two or more "digits".

Reply:
Thanks for spotting that! I'll change it first then do the merge

Reply:
Also we are updating with the screenshot of the property list from apollo's pull request right